src/printer/index.js

```diff
@@ -138,35 +138,61 @@ function printWithItem(path, print) {
   return group(concat(parts));
 }
 
-function printIf(path, print, isElse) {
-  const n = path.getValue();
+function printIf(path, print) {
+  function printOrElse(path) {
+    const n = path.getValue();
+
+    if (n.orelse && n.orelse.length > 0) {
+      if (n.orelse[0].ast_type === "If") {
```

Comment:
This should check that the length is exactly 1. Otherwise you transform

```python
if foo:
    foo()
else:
    if bar:
        bar()
    if baz:
        baz()
```

into

```python
if foo:
  foo()
elif bar:
  bar()
elif baz:
  baz()
```

And other weird things whenever an `if` shows up under an `else` for any reason.

Reply:
Good catch! I've submitted an initial PR to fix this: https://github.com/prettier/prettier-python/pull/20